src/VersionsJSONUtil.jl

```diff
@@ -195,6 +231,7 @@ function main(out_path)
             rm(filepath)
         end
     end
+    @info "Tried $(number_urls_tried) versions, successfully downloaded $(number_urls_success)"
```

Comment:
This feels a bit pointless since we know and expect a large number of version combinations to not be available, so the exact number doesn't mean much. But I don't mind it being there either.